### PR TITLE
include .rpgleinc extension

### PR DESCRIPTION
**Pull request by GajenderI**
.rpgleinc is used for copy books

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -150,11 +150,13 @@
...
                 "aliases": [
                     "RPGLE",
-                    "SQLRPGLE"
+                    "SQLRPGLE",
+                    "RPGLEINC"
                 ],
                 "extensions": [
                     ".rpgle",
-                    ".sqlrpgle"
+                    ".sqlrpgle".
```
Comment (barrettotte) [inline package.json:158]: This should be a comma for valid json

**Comment by barrettotte** [comment]
I agree, I've seen `.rpgleinc` enough that I think it would be useful to include this additional file extension under RPGLE

**Comment by GajenderI** [comment]
@barrettotte typo corrected please merge this

**Comment by barrettotte** [comment]
@GajenderI Change included in https://github.com/barrettotte/vscode-ibmi-languages/releases/tag/v06.13, VS Code marketplace should have the new release available shortly; Thanks!

**Comment by GajenderI** [comment]
Thank you :)

**Comment by chrjorgensen** [comment]
@GajenderI Please note that the `.rpgleinc` suffix is optional and not mandatory - it is only mandated by your company standard. The RPG compiler will take whatever suffix, if any, you specify in the `/COPY` or `/INCLUDE` statement. Only if no suffix is specified and no file is found with the name specified, will the RPG compiler search for a file with the `.rpgleinc` suffix - and also the `.rpgle` suffix.

Some also hase rules for the name of the source files or directories, where copybooks are stored. I've seen companyes use QRPGLEINC and other names. Again, this is not mandated by the RPG compiler - however, if no source file is specified, it will search for a source file called `QRPGLESRC`.

So be careful to distinguish between your company naming standards and the compiler requirements.

For the RPGLE compiler search rules, see here:
https://www.ibm.com/docs/en/i/7.3?topic=directives-copy-include
https://www.ibm.com/docs/en/i/7.3?topic=files-include

To make VS Code treat `.rpgleinc` files exactly the same as `.rpgle` files, you could configure the file association for `.rpgleinc` to be `RPGLE`: Just click on the language in the bottom bar, click `Configure File Association for '.rpgleinc'` and select `RPGLE`. This way you don't have to make every VS Code extension aware of the `.rpgleinc` file suffix...

**Pull request merged**